homeassistant/components/zoneminder/camera.py

```diff
@@ -38,30 +42,20 @@ class ZoneMinderCamera(MjpegCamera):
 
     _attr_should_poll = True  # Cameras default to False
 
-    def __init__(self, monitor, verify_ssl):
+    def __init__(self, monitor: Monitor, verify_ssl: bool) -> None:
         """Initialize as a subclass of MjpegCamera."""
         super().__init__(
             name=monitor.name,
             mjpeg_url=monitor.mjpeg_image_url,
             still_image_url=monitor.still_image_url,
             verify_ssl=verify_ssl,
         )
-        self._is_recording = None
-        self._is_available = None
+        self._attr_is_recording = False
+        self._attr_available = False
```

Comment:
`is_recording` and `available` should not be returning `None`, so I have adjusted to return the equivalent falsy value to stay close to the original implement

However, I wonder if it wouldn't be better to instead use `update_before_add = True` in `add_entities`
```python
    add_entities(cameras, True)
```
Which means we could then drop these two lines.

Reply:
I think the current is fine tbh. Let's someone improve it that can test/uses it.